hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileChecksum.java

```diff
@@ -77,6 +81,31 @@
   private String stripedFile2 = ecDir + "/stripedFileChecksum2";
   private String replicatedFile = "/replicatedFileChecksum";
 
+  private String checksumCombineMode;
+  private boolean expectComparableStripedAndReplicatedFiles;
+  private boolean expectComparableDifferentBlockSizeReplicatedFiles;
```

Comment:
Do we need three boolean fields? It seems that the three booleans have the same value.
Maybe we should simplify the implementation by replacing the three fields by a single field (i.e., `isCompositeCRC`)

Reply:
I removed boolean params and updated issue/PR title.